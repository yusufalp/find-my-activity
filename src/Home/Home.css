.home-header a{
  border: 1px solid black;
  padding: 6px;
  margin: 0 6px;
  text-decoration: none;
  color: black
}