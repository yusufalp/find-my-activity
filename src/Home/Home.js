import React from 'react';
import './Home.css';

export default function Home() {
  return (
    <div>
      <header className='home-header'>
        <h1>Find My Activity</h1>
        <a href='/find'>Find Activity</a>
        <a href='/add'>Add Activity</a>
      </header>
      <section>
        <h2>What is this website?</h2>
        <p>
          Lorem ipsum dolor sit amet consectetur
          adipisicing elit. Accusamus, veniam
          voluptatibus a deleniti ab sequi? At
          consequuntur porro nesciunt et vel ratione
          accusamus voluptatibus, distinctio est minus
          explicabo error ab ut reprehenderit repudiandae
          ullam magni. Quam quasi possimus praesentium harum
          eaque in odit, maiores repellat iste architecto
          magni explicabo tempora necessitatibus eligendi
          saepe totam voluptatum vel. Optio, neque. Voluptas
              </p>
      </section>
      <hr />
      <section>
        <h2>How to use this website?</h2>
        <p>
          Lorem ipsum dolor sit amet consectetur
          adipisicing elit. Accusamus, veniam
          voluptatibus a deleniti ab sequi? At
          consequuntur porro nesciunt et vel ratione
          accusamus voluptatibus, distinctio est minus
          explicabo error ab ut reprehenderit repudiandae
          ullam magni. Quam quasi possimus praesentium harum
          eaque in odit, maiores repellat iste architecto
          magni explicabo tempora necessitatibus eligendi
          saepe totam voluptatum vel. Optio, neque. Voluptas
              </p>
      </section>
    </div>
  )
}