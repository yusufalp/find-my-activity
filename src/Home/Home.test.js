import React from 'react';
import ReactDOM from 'react-dom';
import Home from './Home';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';

describe('Navbar component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(
      <BrowserRouter>
        <Home />
      </BrowserRouter>, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders the UI as expected', () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <Home />
        </BrowserRouter>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
})
