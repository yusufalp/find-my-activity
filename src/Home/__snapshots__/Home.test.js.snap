// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar component renders the UI as expected 1`] = `
<div>
  <header
    className="home-header"
  >
    <h1>
      Find My Activity
    </h1>
    <a
      className="home-links"
      href="/find"
      onClick={[Function]}
    >
      Find Activity
    </a>
    <a
      className="home-links"
      href="/add"
      onClick={[Function]}
    >
      Add Activity
    </a>
  </header>
  <div
    className="group"
  >
    <div
      className="item"
    >
      <img
        alt="timer"
        className="feature-img"
        src="/images/chronometer.png"
      />
      <h3>
        Save time
      </h3>
      <p>
        Sure you can search for many activities online however, Find My Activity is the only place you can find it right away, We do not let you look for the needle in the hay
      </p>
    </div>
    <div
      className="item"
    >
      <img
        alt="group of people"
        className="feature-img"
        src="/images/group.png"
      />
      <h3>
        Give back to community
      </h3>
      <p>
        You can share your favorite activities with our community and we are forever grateful for that. After all, it takes a village, am I right?
      </p>
    </div>
    <div
      className="item"
    >
      <img
        alt="happy family"
        className="feature-img"
        src="/images/happiness.png"
      />
      <h3>
        Enjoy
      </h3>
      <p>
        These activities are meant for kids (and kids inside you). So you can either enjoy it by yourself or the whole family
      </p>
    </div>
  </div>
  <img
    alt="group of kids"
    className="group-kids"
    src="/images/group-kids.png"
  />
  <hr />
  <section>
    <h2>
      How to use this website?
    </h2>
    <p>
      There are two sections in this website where you can interact about the activities: Find and Add Activity.
    </p>
    <p>
      In the Find Activity section, search activities by the category. Choose outdoors if you want your child to work on something outdoor. Do you want something more like academics related, we got that. How about sports? We are constantly increasing the amount of the categories as we see more requests.
    </p>
    <p>
      In the Add Activity section, you can add your own activity. Have you been doing an activity with your child, share it with our community. Simply add the details of the activity requested and it will be posted for our users to search for.
    </p>
    <p>
      Enjoy!
    </p>
  </section>
</div>
`;
